src/auto/injector.js

```diff
@@ -819,7 +818,8 @@ function createInjector(modulesToLoad, strictDi) {
         try {
           path.unshift(serviceName);
           cache[serviceName] = INSTANTIATING;
-          return cache[serviceName] = factory(serviceName, caller);
+          cache[serviceName] = factory(serviceName, caller);
```

Comment:
I've looked again and I probably did it differently above because the LHS expression was more complex so more would have to be repeated.